control-plane/plugin/src/bin/rest-plugin/main.rs

```diff
@@ -62,10 +78,18 @@ async fn execute(cli_args: CliArgs) {
             }
             GetResources::Pools => pool::Pools::list(&cli_args.output).await,
             GetResources::Pool { id } => pool::Pool::get(id, &cli_args.output).await,
-            GetResources::Nodes => node::Nodes::list(&cli_args.output).await,
+            GetResources::Nodes(args) => {
+                if args.show_drain() {
```

Comment:
would be good to move this logic into node::Nodes somehow

Reply:
Agreed, I'll see what I can do.